lib/util.js

```diff
@@ -17,6 +17,8 @@ const get = (url) =>
     }
     protocol
       .get(url, (response) => {
+        const status = response.statusCode;
+        if (status !== 200) return;
```

Comment:
@sebastian-meier Arrrrrrgggghhh you love to write stuff yourself. I get it, but this did throw an unexpected error if the requested file does not exist! 

![](https://media3.giphy.com/media/xT5LMxwQpYU9HqsfcY/giphy.gif)